src/components/commit-list-item.component.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { StyleSheet, TouchableHighlight, View } from 'react-native';
+import { ListItem } from 'react-native-elements';
+import { colors, fonts } from 'config';
+
+type Props = {
+  commit: Object,
+  navigation: Object,
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    paddingRight: 10,
+    paddingVertical: 5,
+    borderBottomWidth: 1,
+    borderBottomColor: colors.greyLight,
+  },
+  listItemContainer: {
+    flex: 1,
+    borderBottomWidth: 0,
+  },
+  title: {
+    color: colors.primaryDark,
+    ...fonts.fontPrimary,
+  },
+});
+
+export const CommitListItem = ({ commit, navigation }: Props) =>
+  <TouchableHighlight
+    onPress={() =>
```

Comment:
Can you please move this arrow function out of this prop? Maybe you can create a function outside of the component and call it here instead.

Reply:
Nice I'm fine either way honestly, but I do see the benefit of having methods separate :+1:

Reply:
Having an arrow function inside a prop may become a perf issue.